src/components/file-upload/file-upload.css

```diff
@@ -1,34 +1,52 @@
-.oui-file-upload-modal {
-    position: absolute;
-    top: 1rem;
-    left: 1rem;
-    bottom: 1rem;
-    right: 1rem;
-    box-sizing: border-box;
-    border: 4px solid blue;
-}
-.oui-file-upload-modal-content {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-}
-.oui-file-upload-modal heading {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
-    text-transform: uppercase;
-    padding: 1rem;
-    background: blue;
-    color: white;
-}
-.oui-file-upload-modal .file-upload-message {
-    text-align: center;
-    display: inline-block;
-    padding: 1rem;
-    background: #f0f0f0;
-    border-radius: 0.3rem;
-}
-.oui-file-upload-modal .oui-documents-icon {
-    margin: 10px auto;
-}
\ No newline at end of file
+oui-file-upload > oui-modal .oui-modal__content {
+  width: 96vw;
+  height: 96vh;
+}
+
+oui-file-upload > oui-modal .oui-modal__content > div,
+  oui-file-upload #oui-file-upload__drop-area {
```

Comment:
I don't think we should use id selectors for css

Reply:
np